app/models/user.rb

```diff
@@ -14,14 +14,15 @@ class User < ActiveRecord::Base
     :twitter_username
   ].freeze
 
+  before_destroy :yank_gems
```

Comment:
I don't think this should change our policy regarding how gems are dealt with. There should always be an owner for a gem. Here's what I think deleting your account should do:

1. For gems with > 1 owner: removes your ownership.
2. For gems with 1 owner (just yourself): yanks all versions of that gem.

We can be very explicit and explain this on a special page as you're deleting your account. It's not an action to be done mistakenly (but that will happen!)

Any objections to this? @dwradcliffe @evanphx @indirect